### PR TITLE
admin team leader add

### PR DESCRIPTION
**Pull request by minhld1709**

### CODE REVIEW

File: app/controllers/admin/teams_controller.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ def show
       format.html
       format.csv {send_data @team.to_csv}
     end
+    @leader = User.find_by id: @team.leader_id
```
Comment (dieunb) [inline app/controllers/admin/teams_controller.rb]: `@leader = @team.leader`

File: app/controllers/teams_controller.rb [hunk 2]
```diff
@@ -7,7 +7,8 @@ def index
     @teams = Team.paginate page: params[:page], per_page: Settings.page.max_page
   end
 
-  def show    
+  def show
+    @leader = User.find_by id: @team.leader_id
```
Comment (dieunb) [inline app/controllers/teams_controller.rb]: ` @leader = @team.leader`

File: app/views/admin/teams/new.html.erb [hunk 3]
```diff
@@ -10,6 +10,12 @@
   </div>
 
   <br/><br/>
+  <div class="field">
+    <%= f.label :leader_id, "Leader" %>
+    <%= f.collection_select :leader_id, User.normal, :id, :name %>
```
Comment (dieunb) [inline app/views/admin/teams/new.html.erb]: try `pluck` to select only `:id, :name`

File: app/views/admin/teams/new.html.erb [hunk 4]
```diff
@@ -10,6 +10,12 @@
   </div>
 
   <br/><br/>
+  <div class="field">
+    <%= f.label :leader_id, "Leader" %>
+    <%= f.select :leader_id, options_for_select(User.normal.pluck(:name, :id)) %>
```
Comment (dieunb) [inline app/views/admin/teams/new.html.erb]: You should set `User.normal.pluck(:name, :id)` at controller

Comment (dieunb) [inline app/views/admin/teams/new.html.erb]: You want to move `User.normal.pluck(:name, :id)` to controller, don't you?

File: app/views/admin/teams/new.html.erb [hunk 5]
```diff
@@ -10,6 +10,12 @@
   </div>
 
   <br/><br/>
+  <div class="field">
+    <%= f.label :leader_id, "Leader" %>
```
Comment (khanhkhanh) [inline app/views/admin/teams/new.html.erb]: i18n

**Pull request merged**